### DISCUSSION TITLE
Why does the page reload when disconnecting Coinbase Wallet?

### DISCUSSION THREAD

**Discussion by ghost** [wevm/wagmi#1662] [Q&A]
This only happens in Coinbase Wallet.

Steps To Reproduce:
```
1. Connect using Coinbase Wallet
2. Reload the page
3. Click disconnect button on the site
```

And after these actions, the page reloads.

**Comment by tmm** [comment]
The `CoinbaseWalletConnector` has an option to disable this behavior `reloadOnDisconnect`, which wagmi defaults to `false`, but it appears there is an issue on the Coinbase Wallet SDK side (see https://github.com/coinbase/coinbase-wallet-sdk/issues/764).

**Reply by ghost** [reply]
Yes, I tried changing `reloadOnDisconnect` = `false`. But it didn't work.
So there is no working way to disable page reloading?

**Reply by tmm** [reply]
Correct, see my previous reply:

> there is an issue on the Coinbase Wallet SDK side (see coinbase/coinbase-wallet-sdk#764).
